### PR TITLE
Add evaluation perf. comparison for t5 & wav2vec tests

### PR DESCRIPTION
**Pull request by vivekgoe**
# What does this PR do?
Addition to test_examples.py to compare evaluation performance with baseline performance. Add evaluation performance targets (as measured on Gaudi) for t5 and wav2vec baseline json files (same to be added for remaining models in future).

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: tests/test_examples.py [hunk 1]
```diff
@@ -56,6 +56,8 @@
 ACCURACY_PERF_FACTOR = 0.99
 # Trainings should last at most 5% longer than the baseline
 TRAINING_TIME_PERF_FACTOR = 1.05
+# Evaluations should last at most 5% longer than the baseline
+EVAL_TIME_PERF_FACTOR = 1.05
```
Comment (regisss) [inline tests/test_examples.py:60]: I think we can just have a single `TIME_PERF_FACTOR` instead of one for training and one for inference.

Comment (vivekgoe) [reply tests/test_examples.py]: Done.

**Comment by vivekgoe** [comment]
@regisss I did run "make style" but it reformats some of the files I did not change. Also, files that are showing formatting failure in CI are not being changed in my patch, any suggestions on what may be going wrong?

**Review by regisss — APPROVED** [review]
LGTM!

**Comment by regisss** [comment]
> @regisss I did run "make style" but it reformats some of the files I did not change. Also, files that are showing formatting failure in CI are not being changed in my patch, any suggestions on what may be going wrong?

Maybe upgrading `black` and `ruff` would help?
```
pip install --upgrade black ruff
make style
```

**Comment by vivekgoe** [comment]
@regisss I upgraded black & ruff, that did not help. But code style check in CI passed once I added other reformatted files (unrelated to my change) also.  Please check and merge if it looks ok now.

**Pull request merged**